Firebase/DynamicLinks/Utilities/FDLUtilities.m

```diff
@@ -227,4 +249,19 @@ BOOL FIRDLMatchesShortLinkFormat(NSURL *URL) {
   return matchMap[serverMatchTypeString] ?: @"none";
 }
 
+void FIRDLAddToWhiteListForCustomDomainsArray(NSArray *_Nonnull customDomains) {
```

Comment:
is the "_Nonnull" needed since there's already NS_ASSUME_NONNULL specified in this file?

Reply:
Done.